File: Models/Artifacts/DestinyArtifactTier.cs
namespace bulkybook.Models;

public class DestinyArtifactTier
{
    public UInt32 tierHash {get;set;}
    public Boolean isUnlocked {get;set;}
    public Int32 pointsToUnlock {get;set;}
    public List<DestinyArtifactTierItem> items {get;set;}
}

File: Scripts/Models/DestinyArtifactTier.ts
import { DestinyArtifactTierItem } from "./DestinyArtifactTierItem";
export interface DestinyArtifactTier
{
    tierHash: number;
    isUnlocked: boolean;
    pointsToUnlock: number;
    items: Array<DestinyArtifactTierItem>;
}